packages/react-tree-grid/package.json

```diff
@@ -5,6 +5,7 @@
   "peerDependencies": {
     "@fluentui/react-components": ">=9.44.4 < 10.0.0",
     "@fluentui/react-shared-contexts": ">=9.7.2 <10.0.0",
+    "@fluentui/react-context-selector": ">=9.1.47 < 10.0.0",
     "@fluentui/keyboard-keys": ">=9.0.6 < 10.0.0",
     "@fluentui/react-tabster": ">=9.14.0 < 10.0.0",
     "@fluentui/react-utilities": ">=9.15.1 < 10.0.0",
```

Comment:
Please declare all of these as dependencies instead: there is no expectation that user will install these packages on their side, but will be forced (or noticed) by a package manager.